### ISSUE TITLE
Request to coordinate timing & reactivate pension/how-to-apply/ page from Pension Benefits Team

### ISSUE THREAD

**Issue by vbellissimo** [department-of-veterans-affairs/va.gov-team#73034]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: `Pension Benefits`
- **OCTO product owner:** `Emily Theis`
- **Product name:** `Pension 527ez form`
- **Product manager:**  `Vicky Bellissimo`
- **Slack channel:** `benefits-pension`
- **Dedicated content writer on your team (if you have one):** `Katherine Fung / katherine-fung`
- **Dedicated a11y specialist on your team (if you have one):** `Unsure`
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [x] Updating existing unauthenticated content on VA.gov
- [x] Something else:  `The Pension 527ez web form did technically exist, but was deactivated on November 8, 2023. We plan on re-launching the web form by the end of this month (January 2024).`

### What's the nature of your initiative and desired outcomes?

The Pension 527ez web form did exist, but was deactivated on November 8, 2023. At the moment, it is not available to users online. We plan on re-launching the web form by the end of this month (January 2024). During the deactivation, a lot of static content that leads to the web form was updated to let users know that the form is now unavailable. We'd like to work with your team to ensure that we revert the content back to the state it was in that allows them to submit the form online. We also need to make sure we can coordinate the % of traffic volume this state of the content/form will be available to.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [x] Yes
- [ ] No
- [ ] I’m not sure

### Where are you at in your timeline?
We are expected to re-launch the Pension 527ez online form by end of January. We anticipate having the state of the content that makes the online form available again to a small amount of people/traffic by January 22. From there, we anticipate increasing the traffic volume incrementally, but exact times frames and volume for each timeframe has not been defined yet.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** #65250

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/pension/527EZ-reactivation-initiative-brief.md)
- [Release plan](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1704481084114/b171b0540054b2844cb7977dd7dd2dfb7aab9c48?sender=u9d01ee38af217e4877e75660)
- [User flow diagram](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1703103128492/30dc0956f99482beffb457eb2ae884430cc2d3f4?sender=u9d01ee38af217e4877e75660)
- [Staging URL for "How to apply" page](https://staging.va.gov/pension/how-to-apply/) that needs to be brought back to the state it was in before deactivation

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+jilladams%2C+RLHecht&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
```

**Comment by vbellissimo** [comment]
@coforma-terry @strelichl @katherine-fung Is this safe to close now that we have the How to Apply page back up? Or should we wait to close until we're at the release to 100% traffic?
